hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm-overview.html

```diff
@@ -110,6 +110,106 @@ <h2>Space Statistics</h2>
     </tbody>
 </table>
 
+<h2>Pipeline Statistics</h2>
+<table class="table table-bordered table-striped">
+    <tbody>
+    <tr>
+        <th>Pipeline State</th>
+        <th>Size</th>
+    </tr>
+    <tr>
+        <td>Closed</td>
+        <td>{{statistics.nodes.data.pipeline.closed}}</td>
+    </tr>
+    <tr>
+        <td>Allocated</td>
+        <td>{{statistics.nodes.data.pipeline.allocated}}</td>
+    </tr>
+    <tr>
+        <td>Open</td>
+        <td>{{statistics.nodes.data.pipeline.open}}</td>
+    </tr>
+    <tr>
+        <td>Dormant</td>
+        <td>{{statistics.nodes.data.pipeline.dormant}}</td>
+    </tr>
+    </tbody>
+</table>
+
+<h2>Container Statistics</h2>
+<table class="table table-bordered table-striped">
```

Comment:
Let's separate this table into two smaller ones: one for "Lifecycle State" and "Health State"

Reply:
I think we could shorten this to just Operational State and Health like how we do in Recon?

Reply:
Sure, let's make it consistent with Recon.